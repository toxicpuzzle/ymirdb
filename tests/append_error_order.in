SET a 1 2 3
APPEND a b 
SET b 2 3
APPEND a b k a
APPEND a b
APPEND a a
APPEND b b a
APPEND b 12sf b
APPEND aa bb
APPEND a b
LIST ENTRIES
bye