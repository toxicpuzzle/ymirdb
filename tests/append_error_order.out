> ok

> no such key

> ok

> no such key

> ok

> not permitted

> not permitted

> no such key

> no such key

> ok

> b [2 3]
a [1 2 3 b b]

> bye
